src/s2mMuscleFatigueState.cpp

```diff
@@ -24,36 +24,47 @@ void s2mMuscleFatigueState::setState(double active, double fatigued, double rest
 {
     // Sanity check for active fibers
     if (active < 0){
+        resting += active;
         s2mError::s2mWarning(0, "Active Fibers Quantity can't be lower than 0, 0 is used then");
+        std::cout <<  "Previous Active Fibers Quantity before set to 0: " << active << std::endl;
```

Comment:
Why did you put a ```cout``` while there is a warning that does the same?
If you want to extend the warning message, you can just extend it. 

The same apply for the next ```cout```!